manifest.json

```diff
@@ -1,39 +1,38 @@
 {
   "manifest_version": 3,
   "name": "Better Youtube Search",
-  "version": "0.0.1",
+  "version": "0.0.2",
   "description": "Improve Youtube search results.",
-   "permissions": ["scripting"],
+  "permissions": ["scripting", "tabs"],
   "icons": {
     "16": "images/play-emoji-16.png",
     "32": "images/play-emoji-32.png",
     "48": "images/play-emoji-48.png",
     "128": "images/play-emoji-128.png"
-},
-"background": {
-    "service_worker": "scripts/background.js"
-},
-"web_accessible_resources": [
-  {
-    "resources": ["css/*.css"],
-    "extension_ids": [
-      "dalpijdkogoelmchapbciodmalgoaoak"
-    ]
+  },
+  "background": {
+    "service_worker": "scripts/background.js",
+    "scripts": ["scripts/background.js"]
+  },
+  "web_accessible_resources": [
+    {
+      "resources": ["css/*.css"],
+      "matches": ["*://www.youtube.com/*"]
     }
   ],
-"content_scripts": [
+  "browser_specific_settings": {
+    "gecko": {
+      "id": "{5748ae93-93bb-4747-a44c-95ff65026309}"
+    }
+  },
+  "content_scripts": [
     {
-        "matches": [
-        "*://www.youtube.com/results"
-      ],
+      "matches": ["*://www.youtube.com/results"],
```

Comment:
I added this because Chrome required two keys in this object in the web_accessible_resources array.